packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -68,7 +68,7 @@ export class NodeSDK {
         tracerProviderConfig.sampler = configuration.sampler;
       }
       if (configuration.traceParams) {
-        tracerProviderConfig.traceParams = configuration.traceParams;
+        tracerProviderConfig.spanLimits = configuration.traceParams;
```

Comment:
the configuration should be changed too
```suggestion
        tracerProviderConfig.spanLimits = configuration.spanLimits;
```